src/dataPipeline.js

```diff
@@ -113,19 +132,21 @@ fluid.data.executePipeline = function (job) {
         if (fluid.hasGrade(grade, "fluid.simpleInputPipe")) {
             var prevOutput = prevKey ? pipeOutputs[prevKey] : {};
             input = onePipe.input === "_" ? prevOutput : pipeOutputs[onePipe.input];
```

Comment:
The usage of `input: "_"` in the config file worths a good explanation. I was very curious when seeing it in the config file until I read this line.

Reply:
This is explained in the README under fluid.simpleInputPipe at https://github.com/inclusive-design/forgiving-data/blob/a24d0899757531d9ddb2fc43192d6a4267305b2c/README.md#fluidsimpleinputpipe